packages/realm/src/Realm.ts

```diff
@@ -283,13 +285,15 @@ export class Realm {
 
   /**
    * Sets the log level.
+   * @param category - TODO: Add docs.
    * @param level - The log level to be used by the logger. The default value is `info`.
    * @note The log level can be changed during the lifetime of the application.
    * @since 12.0.0
    */
-  static setLogLevel(level: LogLevel) {
-    const bindingLoggerLevel = toBindingLoggerLevel(level);
-    binding.Logger.setDefaultLevelThreshold(bindingLoggerLevel);
+  static setLogLevel(category: LogCategory, level: LogLevel) {
+    // TODO: Adapting to new logging in Core. We may want to add an assertion about the
+    //       category as Core will throw if the category is not one of the valid values.
+    binding.LogCategoryRef.getCategory(category).setDefaultLevelThreshold(toBindingLoggerLevel(level));
```

Comment:
I wonder if this could / should happen in a separate PR.

Reply:
I was thinking that too.  This should be focused on Mixed.
This is a pretty large change from core.

Reply:
Yup, I also want it to be focused on Mixed. For this PR, the intent was not to include the logger changes as something to be reviewed here but were needed in order to compile, so that I can only focus on Mixed. There's more info on this in the PR description.

But as Andrew and I talked about, I could try to pull it out to a separate branch, then updating the logger will need a ticket so we can plan it 👍